src/components/screens/Plan/PlanSidebar.tsx

```diff
@@ -64,7 +65,9 @@ export const PlanSidebar = (props: PlanSidebarProps) => {
 		setFrequency(props.plan?.frequency ?? 'som');
 	}, [props.plan?.frequency]);
 
-	const submitPlanSettingsChange = () => {
+	const submitPlanSettingsChange = (e: React.SyntheticEvent) => {
+		e.preventDefault();
+		console.log('submitPlanSettingsChange');
```

Comment:
Remove console.log 😉

Reply:
Rookie Mistake from my end, hahaha, I've removed that, thanks

Reply:
No problem, it happens